e2e-tests/cypress.config.js

```diff
@@ -0,0 +1,28 @@
+const { defineConfig } = require('cypress')
+const cucumber = require('cypress-cucumber-preprocessor').default
+
+module.exports = (on, config) => {
```

Comment:
config not used.

Reply:
the config variable is not used. 

Your IDE / Text editor should be showing this warning. 

No unused vars

Reply:
I use vscode, no warning is showing.

Reply:
I have removed the config for now.

Reply:
the config var is still showing in the PR

Reply:
I tried pushing with no luck.